deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -15,10 +15,11 @@ data:
     @include logs.conf
   buffer.output.conf: |-
     compress gzip
-    flush_interval "#{ENV['FLUSH_INTERVAL']}"
-    flush_thread_count "#{ENV['NUM_THREADS']}"
-    chunk_limit_size "#{ENV['CHUNK_LIMIT_SIZE']}"
-    total_limit_size "#{ENV['TOTAL_LIMIT_SIZE']}"
+    flush_interval "5s"
+    flush_thread_count "4"
```

Comment:
Should these contain double quotes for numeric values as well?

Reply:
I believe quotes are fine for these values, Fluentd parses these parameters in the correct format, based on https://docs.fluentd.org/configuration/config-file#supported-data-types-for-values